### PR TITLE
[SPARK-11333][Streaming]Add executorId to ReceiverInfo and display it in UI

### PR DESCRIPTION
**Pull request by zsxwing**
Expose executorId to `ReceiverInfo` and UI since it's helpful when there are multiple executors running in the same host. Screenshot:

screen shot 2015-11-02 at 10 52 19 am

### CODE REVIEW

File: streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala [hunk 1]
```diff
@@ -402,7 +402,7 @@ private[ui] class StreamingPage(parent: StreamingTab)
         <tr>
           <th style="width: 151px;"></th>
           <th style="width: 167px; padding: 8px 0 8px 0"><div style="margin: 0 8px 0 8px">Status</div></th>
-          <th style="width: 167px; padding: 8px 0 8px 0"><div style="margin: 0 8px 0 8px">Location</div></th>
+          <th style="width: 167px; padding: 8px 0 8px 0"><div style="margin: 0 8px 0 8px">Executor ID / Host</div></th>
```
Comment (zsxwing) [inline streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala]: Changed `Location` to `Executor ID / Host` to make it consistent with the stage page.

**Comment by zsxwing** [comment]
ping @tdas this is a very small PR.

**Comment by tdas** [comment]
Now that JavaStreamingListener has been merged, I think this PR should be updated to expose the executorId in Java as well.

**Comment by zsxwing** [comment]
> Now that JavaStreamingListener has been merged, I think this PR should be updated to expose the executorId in Java as well.

Oh, right!

**Comment by tdas** [comment]
LGTM! Merging this to master and 1.6! Thanks @zsxwing !

**Pull request closed without merging**